Data/models/BraveTodaySourceMO.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import CoreData
+
+public final class BraveTodaySourceMO: NSManagedObject, CRUD {
```

Comment:
wdyt about renaming to something like `ModifiedBraveTodaySources`
atm this class is misleading as it only stores sources that we have manually toggled

and add a comment that full list of sources is stored as a json file

If we want to migrate to using database for full sources list we can keep the name but should add helpful comments

Reply:
Sure! Yeah naming is just based on our original naming scheme. I can change it to reflect its purpose now